server/bin/pbench-index.py

```diff
@@ -330,6 +353,12 @@ def main(options, name):
                         es_res = es_index(
                             idxctx.es, actions, fp, idxctx.logger, idxctx._dbg
                         )
+                except CustomError1:
+                    idxctx.logger.exception("Continuing with next TarBall")
+                    continue
+                except CustomError2:
+                    idxctx.logger.exception("Breaking out of the loop")
+                    break
```

Comment:
Assuming I've got it right, this goes to the `finally` on l.485: is that correct?

Reply:
yeah